RevenueCat/Scripts/StoreProduct.cs

```diff
@@ -62,31 +62,36 @@ public StoreProduct(JSONNode response)
             {
                 DefaultOption = new SubscriptionOption(defaultOptionJsonNode);
             }
+
             var subscriptionOptionsResponse = response["subscriptionOptions"];
-            if (subscriptionOptionsResponse == null)
+            if (subscriptionOptionsResponse == null) 
```

Comment:
we can kill this trailing space. Doesn't matter though

Reply:
That was my favorite space